main copy.py

```diff
@@ -0,0 +1,95 @@
+
+#Homie: Adam Cohen
+#date: 03/12/18
+#part 1 - serching apartments from excel!
+
+import sys
+import xlrd
+from ApartmentAd import *
+
+
+# Assign spreadsheet fileLocation`
+fileName = 'homie_short.xlsx'
+fileLocation = "C:/Users/adam/Desktop/homie/homie_short.xlsx"
+# Give the location of the file
+loc = (fileLocation)
+# To open Workbook
+wb = xlrd.open_workbook(loc)
+sheet = wb.sheet_by_index(0)
+
+#get number of rows and colums
+rows = sheet.nrows
+#print("rows=", rows)
+colums = sheet.ncols
+#print ("colums=", colums)
+
+def check_input_values():
+    debug = 1 #for debug
+    if(len(sys.argv)== 1):#for debug
+        find_suitable_apartment(2, 3, 1799, 1801, 4) #for debug
+        exit(-100)#for debug
+    if(len(sys.argv)!= 6 ):
+        print("wrong amount of inputs")
+        exit(-1)
+    if(int(sys.argv[1])<0):
+        print("minimum number of rooms can't be smaller then 0")
+        exit(-1)
+    if (int(sys.argv[2]) < 0):
+        print("maxmium number of rooms can't be smaller then 0")
+        exit(-1)
+    if (int(sys.argv[2]) < int(sys.argv[1])):
+        print("maximum number of rooms can't be smaller then minimum number of rooms")
+        exit(-1)
+    if (int(sys.argv[3]) < 0):
+        print("minimum price can't be smaller then 0")
+        exit(-1)
+    if (int(sys.argv[4]) < 0):
+        print("minimum price can't be smaller then 0")
+        exit(-1)
+    if (int(sys.argv[4]) < int(sys.argv[3])):
+        print("maximum price can't be smaller then minimum price")
+        exit(-1)
+    if(int(sys.argv[5])<0 or int(sys.argv[5])>3):
+        print("invalid type, must be in the range of 0-3")
+        exit(-1)
+
+
+def find_suitable_apartment(min_rooms, max_rooms, min_price, max_price, required_type):
+    count = 1
```

Comment:
try to split the functions even more, to make the code more modular and easier to read.
for instance, if you found an apartment, then have a different function that prints it to the user